phpunit.xml

```diff
@@ -9,4 +9,9 @@
             <directory suffix=".php">./tests</directory>
         </testsuite>
     </testsuites>
+    <php>
+        <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
+        <env name="DB_CONNECTION" value="testing"/>
+        <env name="APP_DEBUG" value="true"/>
```

Comment:
Is it mandatory to run the tests in debug mode?

Reply:
True. Isn't mandatory.